x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/parser/ExpressionTests.java

```diff
@@ -188,6 +189,101 @@ public void testTripleDoubleQuotedUnescapedString() {
         assertEquals("line 1:52: token recognition error at: '\"'", e.getMessage());
     }
 
+    public void testStringWithIncorrectUnicodeEscapedChars() {
```

Comment:
This test could be broken in multiple sub-tests, each checking the same message on different arguments in a loop instead of repeating the statements.

Reply:
Sure, makes sense